main.js

```diff
@@ -43,9 +57,11 @@
   function setRange() {
     rangeView[0].innerText = minRange.value;
     rangeView[1].innerText = maxRange.value;
-    var min = parseInt(minRange.value)
-    var max = parseInt(maxRange.value)
+    var min = parseInt(minRange.value || 1);
```

Comment:
I don't see why we would need these lines of code.   Isn't the min value already default of 1 and max value of 100?

Reply:
No I hard coded them instead